src/admin/aws/upload.integration.ts

```diff
@@ -8,14 +8,16 @@ import { ApprovedItemS3ImageUrl } from '../../shared/types';
 
 const testFilePath = __dirname + '/test-image.jpeg';
 
+// Port is not used in CircleCI, so make it optional in the regex.
+// Also, AWS Upload turns the test file in CircleCI into a PNG.
```

Comment:
that's strange

Reply:
@kschelonka, yes - this is only happening in CircleCI, on Dev the uploaded image keeps its original format